lib/presentation/screen/analytics_screens/analytics_tab_bar_view.dart

```diff
@@ -36,6 +35,7 @@ class _AnalyticsTabBarViewState extends State<AnalyticsTabBarView>
           leading: PrimaryAppBarItem(
             icon: Icons.arrow_back_ios_outlined,
             label: 'Back',
+            onTap: () => context.goNamed('MainScreen'),
```

Comment:
do not use magic strings. like 'MainScreen'. create static const variables instead and use them.

Reply:
Okay